osgp/protocol-adapter-mqtt/.gitignore

```diff
@@ -0,0 +1,46 @@
+#################
+## Eclipse
+#################
+
+*.pydevproject
+.project
+.metadata
+.gradle
+bin/
+tmp/
+target/
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.classpath
+.settings/
+.loadpath
+
+####################
+# IntelliJ
+####################
+*.iml
+.idea/
+
+# External tool builders
+.externalToolBuilders/
+
+# Locally stored "Eclipse launch configurations"
+*.launch
+
+# CDT-specific
+.cproject
+
+# PDT-specific
+.buildpath
+
+#Spring MVC webapp
+.springBeans
+
+# Eclipse Checkstyle plugin
+.checkstyle
+
+# KDE folder parameters file
+**/.directory
```

Comment:
There's a .gitignore file for the repo: https://github.com/OSGP/open-smart-grid-platform/blob/development/.gitignore
Please remove this .gitignore file, or update the .gitgnore file for the repo.

Reply:
I used the same approach as other protocol adapters which also have a .gitignore file